.travis.yml

```diff
@@ -4,4 +4,5 @@ script:
   - wget https://github.com/broadinstitute/cromwell/releases/download/31/womtool-31.jar
   - for F in *.wdl; do echo $F; java -jar womtool-31.jar validate $F; done
   - 'if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then git submodule foreach --recursive git checkout $TRAVIS_BRANCH && git submodule foreach --recursive git pull; fi'
+  - "git submodule foreach --recursive git status"
```

Comment:
why is this?

Reply:
Oh, right. This should probably be removed. We added this because we weren't sure why the test failed. 
Submodule commit hashes were on develop, but were 'dirty'. Turned out the tasks submodule inside of the submodules hadn't been updated.